### ISSUE TITLE
Feature: Use Rails GlobalID

### ISSUE THREAD

**Issue by henrikbjorn** [abevoelker/devise-passwordless#22]
These can be signed and are encoded completely as base64 also they support expiration out the box.

Consider using that instead of the custom token in the library

**Comment by bilouw** [comment]
Hi,

I used a similar technique with GlobalID, found here: https://www.magicalruby.com/implementing-magic-links-in-rails/

My problem with this technique is that magic-link can't be revoke after use. There is no way (that i know) to expire an GlobalID by hand.

So i took this gem especially for this feature: **magic link are usable only once**
**This is a very important point for the security of magic links**

If you use GlobalID in the future, how do you plan to handle magic link expiration ?

**Comment by abevoelker** [comment]
This feature is now available on the master branch and will be the default tokenization method for new installations. It will be released with v1.0 of the gem, hopefully this weekend. You can try it out early until then:

```ruby
gem "devise-passwordless", github: "abevoelker/devise-passwordless"
```

Upgrade guide for going from 0.x to 1.0: https://github.com/abevoelker/devise-passwordless/blob/master/UPGRADING.md
